e2e-tests/scheduled-backup/run

```diff
@@ -7,13 +7,13 @@ test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
```

Comment:
please roll back the file permissions.

Reply:
Resolved in 9621b66f1ebeeb2c13c2c436d2b89f49332f22a0